process_pr.py

```diff
@@ -138,7 +138,7 @@ def format(s, **kwds):
 REGEX_IGNORE_COMMIT_COUNT = "\+commit-count"
 TEST_WAIT_GAP = 720
 ALL_CHECK_FUNCTIONS = None
-EXTRA_RELVALS_TESTS = ["threading", "gpu", "high-stats", "nano"]
+EXTRA_RELVALS_TESTS = ["threading", "gpu", "high-stats", "nano", "hlt_p2_timing"]
```

Comment:
it is not a relval test, so please remove it from here and add it in `EXTRA_TESTS` line below